pkg/controller/account/ec2.go

```diff
@@ -129,10 +129,11 @@ func (r *ReconcileAccount) BuildandDestroyEC2Instances(reqLogger logr.Logger, aw
 
 // CreateEC2Instance creates ec2 instance and returns its instance ID
 func CreateEC2Instance(reqLogger logr.Logger, client awsclient.Client, ami string) (string, error) {
-	// Create EC2 service client
 
+	// Retain instance id
 	var instanceID string
```

Comment:
This should be 

`instanceID  = ""`

that way if we never action it and return it it will not return a nil variable , the other option is to return an empty string instead of the the variable when we return an error.